packages/playwright/src/worker/workerMain.ts

```diff
@@ -103,13 +105,18 @@ export class WorkerMain extends ProcessRunner {
   }
 
   override async gracefullyClose() {
+    if (this._isMissingProject) {
```

Comment:
Let's instead move this to the line 115 (after calling `this._stop()`), and perhaps check `!this._config` which is equivalent to not have loaded once.